gin_integration_test.go

```diff
@@ -115,17 +115,17 @@ func TestWithHttptestWithAutoSelectedPort(t *testing.T) {
 	testRequest(t, ts.URL+"/example")
 }
 
-func TestWithHttptestWithSpecifiedPort(t *testing.T) {
-	router := New()
-	router.GET("/example", func(c *Context) { c.String(http.StatusOK, "it worked") })
-
-	l, _ := net.Listen("tcp", ":8033")
-	ts := httptest.Server{
-		Listener: l,
-		Config:   &http.Server{Handler: router},
-	}
-	ts.Start()
-	defer ts.Close()
-
-	testRequest(t, "http://localhost:8033/example")
-}
+// func TestWithHttptestWithSpecifiedPort(t *testing.T) {
```

Comment:
We got 2 options, just continue with `tip` which seems to be a little bit more stable than `master` or accept errors on master as this looks like an upstream issue.